inc/options/objectcache.php

```diff
@@ -116,5 +116,58 @@
 		</table>
 
 		<?php Util_Ui::postbox_footer(); ?>
+
+		<?php Util_Ui::postbox_header( esc_html__( 'Purge Via WP Cron', 'w3-total-cache' ), '', 'objectcache_wp_cron' ); ?>
+		<table class="form-table">
+			<?php
+			$c           = Dispatcher::config();
+			$wp_disabled = ! $c->get_boolean( 'objectcache.wp_cron' );
+
+			Util_Ui::config_item(
+				array(
+					'key'            => 'objectcache.wp_cron',
+					'label'          => esc_html__( 'Enable WP-Cron Event', 'w3-total-cache' ),
+					'checkbox_label' => esc_html__( 'Enable', 'w3-total-cache' ),
+					'control'        => 'checkbox',
+					'description'    => esc_html__( 'Enabling this will schedule a WP-Cron event that will flush the Object Cache. If you prefer to use a system cron job instead of WP-Cron, you can schedule the following command to run at your desired interval: "wp w3tc flush object".', 'w3-total-cache' ),
+				)
+			);
+
+			$time_options = array();
+			for ( $hour = 0; $hour < 24; $hour++ ) {
+				foreach ( array( '00', '30' ) as $minute ) {
+					$time_value                  = $hour * 60 + intval( $minute );
+					$time_label                  = gmdate( 'g:i a', strtotime( sprintf( '%02d:%s', $hour, $minute ) ) );
```

Comment:
The config values should be in UTC/GMT, but the label in locale time -- use https://developer.wordpress.org/reference/functions/get_date_from_gmt/

Reply:
Updated. Not sure if the changes satisfy